src/components/PostList.tsx

```diff
@@ -1,9 +1,37 @@
 import React from "react";
+import { useAppSelector } from "../redux/hooks";
+import { FeedItem } from "../utilities/types";
+import { NoteActivityPub } from "../utilities/activityPubTypes";
 
 const PostList: React.FC = () => {
+  const feed: FeedItem[] = useAppSelector((state) => state.feed.feed);
+  feed.filter((post) => post?.content?.type === "Note");
+
+  const postItems = feed.map((post: FeedItem, index: number) => {
+    const noteContent: NoteActivityPub = post.content as NoteActivityPub;
+    return (
+      <div key={index}>
+        {noteContent && (
+          <>
+            <div>{post.fromAddress}</div>
+            <div>{noteContent.content}</div>
+            {noteContent.attachment && (
+              <img
+                width={400}
+                src={noteContent.attachment[0]?.url}
+                alt="undefined"
```

Comment:
Why does `alt="undefined"` here? I would have expected either `""` or perhaps `=noteContent.attachment[0]?.url ? "Image for post" : ""` or something else. https://www.a11yproject.com/posts/2013-01-14-alt-text/